app/controllers/places_controller.rb

```diff
@@ -1,10 +1,12 @@
 class PlacesController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Cette ligne signifie bien que le user n'a pas besoin de se log in avant de réserver ?
Il faudra test le flux pour voir à quel moment on claque le log in obligatoire

Reply:
Oui je vais rajouter les infos dans la navbar very soon 👀

Reply:
Avec un petit environnement de test on pourrait dispatch les tests ça serait stylé.
Je demanderai aux profs de samedi comment set up un bac à sable (pour notre culture, pas forcement utile pour nos projets)